developer-docs-site/docs/guides/move-guides/index.md

```diff
@@ -15,8 +15,7 @@ Start here to learn how the Move language works on the Aptos blockchain.
 ## Aptos Move documentation
 
 Then review our auto-generated contents for more Move documentation.
-
-* [Aptos tokens](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/doc/overview.md)
-* [Aptos framework](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/doc/overview.md)
-* [Aptos stdlib](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/doc/overview.md)
-* [Move stdlib](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/move-stdlib/doc/overview.md)
+* *Aptos tokens* - [main](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/doc/overview.md), [testnet](https://github.com/aptos-labs/aptos-core/blob/testnet/aptos-move/framework/aptos-token/doc/overview.md), [devnet](https://github.com/aptos-labs/aptos-core/blob/devnet/aptos-move/framework/aptos-token/doc/overview.md)
```

Comment:
Should we have `mainnet` instead of `main`?

Reply:
yeah I agree with greg, `mainnet` is clearer - when I said `main` people usually asked me to clarify if I meant `github main` or `mainnet`

Reply:
ahhh but looking at this link it does mean `github main`? bc what's on `github main` is not necessarily on `mainnet` yet?

Reply:
It's because we don't have the mainnet docs in place yet:
https://github.com/aptos-labs/aptos-core/blob/mainnet/aptos-move/framework/aptos-token/doc/overview.md

Perhaps @wrwg can advise on when. Likely the next mainnet release, I imagine.

Reply:
Related, do we describe our branching strategy and flow anywhere, meaning `main` as upstream and then:
main > devnet > testnet > mainnet

But I am not sure.

Reply:
Good point. I phrased as:

"Please note, the `mainnet` versions of the documents below are not yet available. The `main` branch below shows the latest development upstream and likely contains functionality not yet available in a *net release."

WDYT? Thanks!